independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanGenerator.java

```diff
@@ -1792,15 +1827,20 @@ void implementCreateForClassBean(ClassOutput classOutput, ClassCreator beanCreat
                                         callback.name()));
                     }
                     reflectionRegistration.registerMethod(callback);
-                    create.invokeStaticMethod(MethodDescriptors.REFLECTIONS_INVOKE_METHOD,
-                            create.loadClass(callback.declaringClass().name().toString()),
-                            create.load(callback.name()), create.newArray(Class.class, create.load(0)), instanceHandle,
-                            create.newArray(Object.class, create.load(0)));
+                    postConstructsBytecode.invokeStaticMethod(MethodDescriptors.REFLECTIONS_INVOKE_METHOD,
+                            postConstructsBytecode.loadClass(callback.declaringClass().name().toString()),
+                            postConstructsBytecode.load(callback.name()),
+                            postConstructsBytecode.newArray(Class.class, postConstructsBytecode.load(0)), instanceHandle,
+                            postConstructsBytecode.newArray(Object.class, postConstructsBytecode.load(0)));
                 } else {
-                    create.invokeVirtualMethod(MethodDescriptor.of(callback), instanceHandle);
+                    postConstructsBytecode.invokeVirtualMethod(MethodDescriptor.of(callback), instanceHandle);
                 }
             }
         }
+        if (postConstructsBytecode != create) {
```

Comment:
The [same](https://github.com/quarkusio/quarkus/pull/32885/files#r1178693156) applies here...

Reply:
Done.